app/uk/gov/hmrc/helptosave/connectors/ApiTwentyFiveCConnector.scala

```diff
@@ -43,11 +45,8 @@ class ApiTwentyFiveCConnectorImpl extends ServicesConfig  with ApiTwentyFiveCCon
 
   private val http = WSHttp
 
-  def getAwards(nino: String)(implicit hc: HeaderCarrier): Future[List[Award]] =
-    http.GET(s"$helpToSaveStubURL/${serviceURL(nino)}").flatMap {
-      _.json.validate[ApiTwentyFiveCValues] match {
-        case JsSuccess(result, _) ⇒ Future.successful(result.awards)
-        case JsError(e) ⇒ Future.failed[List[Award]](new Exception("Could not parse awards " + e.head))
-      }
+  def getAwards(nino: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Result[List[Award]] =
+    Result(http.get(s"$helpToSaveStubURL/${serviceURL(nino)}")).subflatMap{ response ⇒
+      response.parseJson[ApiTwentyFiveCValues].map(_.awards)
```

Comment:
`EitherT[Future,String,A]` wraps a `Future[Either[String,A]]` - there are various `flatMap`s which let you map into the `EitherT` and allow you to use the various layers of the data structure - straightforward `flatMap` has the type signature `flatMap(f: A => EitherT[Future,String,B]): EitherT[Future,String,B]` as expected, `subflatMap` has type signature `subflatMap(A => Either[String,B]): EitherT[Future,Stirng,B]` which let's you return an `Either` in 'f' without having to explicitly wrapping it in an `EitherT`. Compare:

```scala
def f(i: Int): Either[String,Long] = ???
val x: EitherT[Future,String,Int] = ???
val y = x.flatMap(i => EitherT.fromEither[Future](f(i)))
```
with
```scala
val y = x.subflatMap{(f)
```